app.js

```diff
@@ -1,35 +1,102 @@
 // Task 1 - Notes for JS Evaluation Predictions
 
 // List 3 expressions that surprised you or gave unexpected results, why?
-
+//2+"pizza" -> "2pizza" was surprising. did not expect 2 to be added to the string
+// 10%2 === 0 -> true initially thought it would be false because 10 divided by 2 equals 5.
+// "science".indexOf("x") === -1 -> true. it surprised me as that is the first time I have seen a negative index position 
 // List 2 exprsssions you felt comfortable interpreting at a glance.
-
+//"new york"[0]->"n"
+// true && false -> false
 // List 1 expression you are still 'shaky' on, why?
-
+// 101%10 -> 1 I understand it is the remainder, however if I was on a calculator, not a computer this would not be the output
 
 
 
 
 // Task 2 - Practice JS Basics
 // 1 - 
-
+/*let idx="abcde".indexOf("d");
+idx = idx+11;
+console.log(idx);//14
+idx * 2;
+console.log(idx);//14
+*/
 // 2 - 
-
+/*let num = 33;
+let isEven = num % 2 ===0;
+console.log(isEven); // false
+console.log(!isEven); // true */
 // 3 - 
-
+/*let str1 = "marker";
+let num = "whiteboard".length - str1.length;
+console.log(num); // 4
+let str2 = "bootcamp";
+console.log(str2[num].toUpperCase()); // "C"
+let char = str2[num].toLowerCase();
+console.log(char + "!"); //"c!" */
 // 4 - 
-
-// 5 - 
-
+/*let sentence = "welcome to bootcamp prep";
+let lastChar = sentence[sentence.length-1];
+console.log(lastChar); //p
+console.log(sentence.indexOf(lastChar));//18
+*/
+// 5 -
```

Comment:
It looks like starting here, you stopped guessing what you would see in the console. It's fine if you guess wrong, just take your best guess!